### PR TITLE
Fix NYTimes first few paragraph missing

### PR DESCRIPTION
**Pull request by sywhb**
The first few paragraphs are deleted by readability because there are too many links in them but these links are not advertisements so I added the class name of the element of the paragraphs to the skipping list when cleaning the nodes.

Test case added too

### CODE REVIEW

File: packages/readabilityjs/Readability.js [hunk 1]
```diff
@@ -2782,14 +2785,15 @@ Readability.prototype = {
           return false;
         }
 
+        var parentClasses = node.parentNode.classList;
```
Comment (jacksonh) [inline packages/readabilityjs/Readability.js:2788]: Can this be null?

Comment (sywhb) [reply packages/readabilityjs/Readability.js:2788]: Nice catch! Thank you

File: packages/readabilityjs/Readability.js [hunk 2]
```diff
@@ -2782,14 +2785,15 @@ Readability.prototype = {
           return false;
         }
 
+        var parentClasses = node.parentNode?.classList || [];
```
Comment (sywhb) [inline packages/readabilityjs/Readability.js:2788]: @jacksonh Default `parentClasses` is an empty array now

**Review by jacksonh — APPROVED** [review]
Will have to make sure we do some testing on demo.

**Pull request merged**